src/status_im2/contexts/quo_preview/navigation/bottom_nav_tab.cljs

```diff
@@ -36,7 +37,15 @@
                :value :unread-dot}]}
    {:label "Counter Label"
     :key   :counter-label
-    :type  :text}])
+    :type  :text}
+
+   {:label   "Customization color"
+    :key     :customization-color
+    :type    :select
+    :options (map (fn [[k _]]
+                    {:key   k
+                     :value (string/capitalize (name k))})
+                  colors/customization)}])
```

Comment:
Since the component now supports customization color, it has been added as a property to the preview screens